sys-cluster/reframe/reframe-3.9.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6..10} )
+
+inherit distutils-r1
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+
+DESCRIPTION="ReFrame is a framework for writing regression tests for HPC systems"
+HOMEPAGE="https://reframe-hpc.readthedocs.io/
+	https://github.com/eth-cscs/reframe/"
+SRC_URI="https://github.com/eth-cscs/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+SLOT="0"
+
+RDEPEND="dev-python/argcomplete[${PYTHON_USEDEP}]
+	dev-python/jsonschema[${PYTHON_USEDEP}]
+	dev-python/lxml[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-5.4.1[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/semver[${PYTHON_USEDEP}]
+	>=sys-apps/archspec-0.1.2"
```

Comment:
@vkarak @victorusu Is the exact version restriction at https://github.com/eth-cscs/reframe/blob/e1487793526d7cf1f51ae088d143f13483c3a244/requirements.txt#L1 there for a specific reason, or just to be on the safe side?
Would archspec 0.1.3 also work?

Reply:
No, it's not... It's just from our laziness to bump our requirements...

Reply:
OK, thanks for the feedback.

We'll assume `archspec` 0.1.3 will work then (and if not, we'll report the trouble we see)